index.html

```diff
@@ -1,55 +1,247 @@
-                <li class="nav-item"><a href="#" class="nav-link">Contact</a></li>
-            </ul>
-            <div class="handburger">
-                <span class="bar"></span>
-                <span class="bar"></span>
-                <span class="bar"></span>
-            </div>
-        </nav>
+      <nav class="navbar">
+        <a href="#">
+          <h2 class="nav-title">Welcome</h2>
+        </a>
+        <ul class="nav-menu">
+          <li class="nav-item"><a href="#" class="nav-link">Portfolio</a></li>
+          <li class="nav-item"><a href="#" class="nav-link">About</a></li>
+          <li class="nav-item"><a href="#" class="nav-link">Contact</a></li>
+        </ul>
+        <div class="handburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+      </nav>
     </header>
 
     <section class="first-section">
-        <div class="container">
-            <h1>Hey there. I’m Daniel</h1>
-            <h1 class="desc">I’m a software developer</h1>
-            <p> I can help you build a product , feature or website Look through some of my work and experience! If you
-                like what you see and have a project you need coded,
-                don’t hestiate to contact me.</p>
-                <ul class="social">
-                    <li><a href="#"><img src="./Images//twitter.svg" alt=""></a></li>
-                    <li><a href="#"> <img src="./Images//linkedin.svg" alt=""></a></li>
-                    <li><a href="#"><img src="./Images/medium.svg" alt=""></a></li>
-                    <li><a href="#"><img src="./Images//github.svg" alt=""></a></li>
-                    <li><a href="#"><img src="./Images//angellist.svg" alt=""></a></li>
-                </ul>
-        </div>
+      <div class="container">
+        <h1>Hey there. I’m Daniel <br> <span class="desc">I’m a software developer</span> </h1>
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <ul class="social">
+          <li>
+            <a href="#"><img src="./Images//twitter.svg" alt="" /></a>
+          </li>
+          <li>
+            <a href="#"> <img src="./Images//linkedin.svg" alt="" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./Images/medium.svg" alt="" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./Images//github.svg" alt="" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./Images//angellist.svg" alt="" /></a>
+          </li>
+        </ul> 
+      </div>
     </section>
-    <!-- <button><img src="./Images//disabled.svg" alt=""></button>
+    <button ><img class="down-icon" src="./Images//disabled.svg" alt="" /></button>
```

Comment:
- [ ] Please remove the `margin-left: 80px;` from this element with class **.down-icon**. This is causing the down-icon to align unevenly to the right.